src/game-loop/src/components/AnimationComponent.cpp

```diff
@@ -47,3 +49,15 @@ void AnimationComponent::start(std::size_t start_frame, std::size_t end_frame, u
     _running = true;
     _finished = false;
 }
+
+void AnimationComponent::start_continued(std::size_t start_frame, std::size_t end_frame, uint32_t time_per_frame_ms, bool loop)
+{
+    _current_frame_index = start_frame + _end_frame_index - _current_frame_index;
```

Comment:
Split this into calculating absolute current frame index (starting from 0) and adding it to the `start_frame`, or at least add a comment that this line is doing it.